elk.yml

```diff
@@ -1,44 +1,52 @@
 # vim: ft=ansible
 ---
 - hosts: elasticsearch
+  become: true
```

Comment:
There are a few of these I had to add. Not sure why, but was running into issues in a Vagrant environment. Think it might have had to do with the OpenStack default user having more open permissions than the vagrant user.